app/src/main/java/simform/gitexcercise/android/validation/Validator.kt

```diff
@@ -0,0 +1,31 @@
+package simform.gitexcercise.android.validation
+
+object Validator {
+
+    object Password {
+        const val MIN_LENGTH = 8
+        const val MAX_LENGTH = 16
+    }
+
+    enum class ValidLogin {
+        VALID, INVALID_EMAIL, INVALID_PASSWORD
+    }
+
+    fun isValidLogin(email: String, password: String): ValidLogin {
```

Comment:
Function name should have meaningful name
Are we authenticating the login?

As per my point of view, we are validating the login credential feilds.

Reply:
okay i have changed the name accordingly